project/index.mjs

```diff
@@ -484,6 +484,32 @@ router.post('/:projectId/removeCustomRoles', auth0Middleware(), async (req, res)
   }
 })
 
+// Update Project Layers
+router.route("/:projectId/layers").put(auth0Middleware(), async (req, res) => {
```

Comment:
https://github.com/CenterForDigitalHumanities/TPEN-interfaces/issues/90#issuecomment-2741432306 This comment shares what I think is a good pattern to provide here.